frontend/main-app/src/components/Login/Login.tsx

```diff
@@ -17,7 +17,16 @@ export const Login = (): React.JSX.Element => {
 
   const [fieldErrors, setFieldErrors] = useState<LoginFormError | null>(null);
   const [errorAlert, setErrorAlert] = useState(authError);
+  const [showErrorAlert, setShowErrorAlert] = useState(!!authError);
 
+  useEffect(() => {
+    if (errorAlert === "authError" && showErrorAlert) {
+      const timeout = setTimeout(() => {
+        setShowErrorAlert(false);
+      }, 3500);
+      return () => clearTimeout(timeout);
+    }
+  }, [errorAlert, showErrorAlert]);
```

Comment:
We can do this without showErrorAlert, making the code a little more concise.

```suggestion
  const { authError } = useParams();
  const navigate = useNavigate();
  const { t } = useTranslation();

  const [fieldErrors, setFieldErrors] = useState<LoginFormError | null>(null);
  const [errorAlert, setErrorAlert] = useState(authError);

  useEffect(() => {
    if (errorAlert === "authError") {
      const timeout = setTimeout(() => {
        setErrorAlert(null);
      }, 3500);
      return () => clearTimeout(timeout);
    }
  }, [errorAlert]);
```